avm/src/main.rs

```diff
@@ -61,69 +64,90 @@ fn main() {
         .arg_from_usage("<AGZ_FILE> 'Specifies the .agz file to deploy'")
       )
     )
+    .subcommand(SubCommand::with_name("daemon")
+      .about("Run an .agz file in daemon mode. Used on deploy within cloud provider VMs.")
+      .arg_from_usage("<APP_ID> 'Specifies the alan app to upgrade'")
+      .arg_from_usage("<AGZ_FILE> 'Specifies the .agz file to deploy'")
+      .arg_from_usage("<DEPLOY_TOKEN> 'Specifies the .agz file to deploy'")
+    )
     .arg_from_usage("[SOURCE] 'Specifies a source ln file to compile and run'");
 
   let matches = app.clone().get_matches();
 
-  match matches.subcommand() {
-    ("run",  Some(matches)) => {
-      let agc_file = matches.value_of("FILE").unwrap();
-      let fp = &format!(
-        "{:}/{:}",
-        env::current_dir().ok().unwrap().to_str().unwrap(),
-        agc_file
-      );
-      exec(&fp, false);
-    },
-    ("compile",  Some(matches)) => {
-      let source_file = matches.value_of("INPUT").unwrap();
-      let dest_file = matches.value_of("OUTPUT").unwrap();
-      std::process::exit(compile(&source_file, &dest_file, false));
-    },
-    ("install",  _) => {
-      let source_file = ".dependencies.ln";
-      if Path::new(source_file).exists() {
-        std::process::exit(compile_and_run(source_file));
-      } else {
-        println!(
-          "{} does not exist. Dependencies can only be installed for {}",
-          source_file, source_file
+  let rt = Builder::new_multi_thread()
```

Comment:
Creating the tokio runtime at the beginning of the CLI now so we can use async everywhere